Docs/MoltenVK_Runtime_UserGuide.md

```diff
@@ -325,6 +325,7 @@ In addition to core *Vulkan* functionality, **MoltenVK**  also supports the foll
 - `VK_KHR_dynamic_rendering`
 - `VK_KHR_fragment_shader_barycentric`
   - *Requires Metal 2.2 on Mac or Metal 2.3 on iOS.*
+- `VK_KHR_format_feature_flags2`
```

Comment:
Close!  😉 Can you just bump it up above `VK_KHR_fragment_shader_barycentric` to keep it alpha in the `KHR` section.

This odd-ish ordering is to match the same `KHR->EXT->vendor->then-name-alpha` ordering used in the Vulkan spec.

Reply:
ah oops I think I might've read over the "r" and thought it was the a... Will fix.